tools/admin/README-NEXT.md

```diff
@@ -0,0 +1,126 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Administrative Operations
+
+The `wskadmin-next` utility is handy for performing various administrative operations against an OpenWhisk deployment.
+It allows you to create a new subject, manage their namespaces, to block a subject or delete their record entirely.
+
+This is a scala based implementation of `wskadmin` utility and is meant to be DB agnostic.
+
+### Build
+
+To build the tool run
+
+    $./gradlew :tools:admin:build
+
+This would create a jar at `tools/admin/build/libs/openwhisk-admin-tools-1.0.0-SNAPSHOT-cli.jar` and install it as an executable script at
+`bin/wskadmin-next`.
+
+### Setup
+
+Build task would have created an executable ay `bin/wskadmin-next`. This script would require config related to `ArtifactStore`
+for accessing database. For e.g. to access user details from default CouchDB setup create a file `application-cli.conf`.
+
+    include classpath("application.conf")
+
+    whisk {
+      couchdb {
+        protocol = "http"
+        host     = "172.17.0.1"
+        port     = "5984"
+        username = "whisk_admin"
+        password = "some_passw0rd"
+        provider = "CouchDB"
+        databases {
+          WhiskAuth       = "whisk_local_subjects"
+          WhiskEntity     = "whisk_local_whisks"
+          WhiskActivation = "whisk_local_activations"
+        }
+      }
+    }
+
+And pass that to command via `-c` option.
+
+    $./wskadmin-next -c application-cli.conf user get guest
```

Comment:
as a future enhancement, making this simpler to pass a canonical conf file (ie dont have to pass it on every command) would be nice. Wether the values are defined in the environment, or there's a canonical conf file one can use.

Reply:
examples below don't show a `-c` file - is one required?

Reply:
Added them for now. Would be removed once we support a file at specific location